python/unitytrainers/models.py

```diff
@@ -159,7 +199,8 @@ def create_new_obs(self, num_streams, h_size, num_layers):
             final_hiddens.append(final_hidden)
         return final_hiddens
 
-    def create_recurrent_encoder(self, input_state, memory_in, name='lstm'):
+    @staticmethod
+    def create_recurrent_encoder(input_state, memory_in, sequence_length, name='lstm'):
         """
```

Comment:
For uniformity, maybe the name in all those methods should be an optional argument (same with reuse) ?